src/lib/parse/regex.js

```diff
@@ -15,8 +15,8 @@ export var matchOffset    = /Z|[+-]\d\d:?\d\d/gi; // +00:00 -00:00 +0000 -0000 o
 
 export var matchTimestamp = /[+-]?\d+(\.\d{1,3})?/; // 123456789 123456789.123
 
-// any word (or two) characters or numbers including two/three word month in arabic.
-export var matchWord = /[0-9]*['a-z\u00A0-\u05FF\u0700-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]+|[\u0600-\u06FF\/]+(\s*?[\u0600-\u06FF]+){1,2}/i;
+// any word (or two) characters or numbers including two/three word month in arabic and words two-three word month in klingon.
+export var matchWord = /(tera’ )*(jar )+([a-z’ ]+|(\s*?[a-z’]+))|[0-9]*['a-z\u00A0-\u05FF\u0700-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]+|[\u0600-\u06FF\/]+(\s*?[\u0600-\u06FF]+){1,2}/i;
```

Comment:
Hmmm...  This is interesting, but I would hope we could address it in a way that doesn't creep locale-specific words into the core.  Even having exceptions for Arabic here feels wrong to me.

@ichernev - Should we put another function into to locale interface so cases like these can be in the locale file instead of here?